test-utils/src/lib.rs

```diff
@@ -0,0 +1,8 @@
+#[cfg(test)]
+mod tests {
+    #[test]
+    fn it_works() {
+        let result = 2 + 2;
+        assert_eq!(result, 4);
+    }
+}
```

Comment:
The license header is missing. Additionally, I suppose something else is expected to be placed in this file. Can we remove it if it isn't needed yet?

Reply:
My bad! Forget to remove it!